chapter07/spg/exercise1.pl

```diff
@@ -0,0 +1,75 @@
+#!perl
+
+use strict;
+use warnings;
+use utf8;
+
+use File::Find;
+use Time::Local qw( timelocal_nocheck );
+
+my $target_dow = $ARGV[0];        # Sunday is 0, Monday is 1, ...

```

Comment:
I thought about doing this, but decided not to since my submission is so late.  I'd also have put some validation to make sure the ARG is a) present; and b) in range.